ambari-admin/src/main/resources/ui/admin-web/app/views/userManagement/groupEdit.html

```diff
@@ -34,7 +34,7 @@
     <div class="form-group">
       <label class="col-sm-2">{{group.groupTypeName | translate}} {{'groups.members' | translate}}</label>
       <div class="col-sm-10">
-        <editable-list items-source="group.editingUsers" resource-type="User" editable="group.group_type == 'LOCAL'"></editable-list>
+        <editable-list items-source="group.editingUsers" resource-type="User" editable="group.group_type == 'LOCAL' || group.group_type == 'LDAP'"></editable-list>
```

Comment:
Are we sure that we want a user to be able to do this?  Any user changes will be potentially reverted when the an LDAP sync operation occurs.